container/common/fsHandler.go

```diff
@@ -78,27 +77,31 @@ func (fh *realFsHandler) update() error {
 		rootDiskErr, rootInodeErr, extraDiskErr error
 	)
 	// TODO(vishh): Add support for external mounts.
+	methodTimeout := fh.period
```

Comment:
so the default period is 1 minute, and we are trying to do 3 operations within that time, whereas the old timeout was 2 minutes for each operation.  Seems like a huge change...